docs/UserGuide.md

```diff
@@ -336,20 +363,35 @@ Retrieve information on a group of profiles at once with this function! This can
 
```

Comment:
perhaps can change it to Export view to CSV file instead of current view

also perhaps can change retrieve information on a group of profiles.... to retrieve information on all profiles.